src/main/java/com/knu/buga1chuk/service/PersonGenerator.java

```diff
@@ -0,0 +1,80 @@
+package com.knu.buga1chuk.service;
+
+import com.knu.buga1chuk.serialization.Person;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class PersonGenerator {
+    private static final Random random = new Random();
+    private static int CURRENT_PERSON_ID = 0;
+
+    public static Person getRandomPerson() {
+
+        CURRENT_PERSON_ID++;
+        String name = generateRandomName();
+        int age = random.nextInt(88) + 12;
```

Comment:
Strange and unreadable.
Could you extract it into the separate method?
The method should receive two parameters: start and end.

Reply:
fix